submissions/ebojudahCherish/html-forms/signUp.html

```diff
@@ -1,88 +1,146 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8">
-        <title>Sign Up</title>
-        <link rel="stylesheet" href="../fontawesome-free-5.15.1-web/css/all.min.css">
-        <link rel="stylesheet" href="../css/styles.css">
-    </head>
 
-    <body>
-        <!-- NAVBAR -->
-        <div class="navbar">
-            <div class="container flex">
-                <h1 class="logo">Beautify.</h1>
-                <nav>
-                    <ul>
-                        <li><a href="../html/productPage.html">Product Page</a></li>
-                        <li><a href="../html-forms/signUp.html">Sign Up</a></li>
-                        <li><a href="../html-forms/contactForm.html">Contact Us</a></li>
-                    </ul>
-                </nav>
+<head>
+    <meta charset="UTF-8">
+    <title>Sign Up</title>
+    <link rel="stylesheet" href="../css/styles.css">
+</head>
+
+<body>
+    <!-- NAVBAR -->
+        <div class="flex justify-between bg-primary px-2 txt-white">
+            <h1>Beautify.</h1>
+            <div class="py-1">
+                <a class="px-1" href="../html/productPage.html">Product Page</a>
+                <a class="px-1" href="../html-forms/signUp.html">Sign Up</a>
+                <a class="px-1" href="../html-forms/contactForm.html">Contact Us</a>
             </div>
         </div>
-    
-        <section class="welcome-text">
-            <div class="container grid">
-                <div class="register-txt">
-                    <h1 class="register-header">Register your Account</h1>
-                    <p class="register-paragraph">Registering an account gives you access to sweet promo and events available to only registered customers</p>
-                    <br>
-                    <p class="register-question">Already got an acount? <a href="#">Sign in</a></p>
-                </div>
 
-                <div class="register-form">
-                    <form action="#" method="POST" class="sg-form">
+    <section class="welcome-text">
+        <div class="container grid">
+            <div class="register-txt">
+                <h1 class="register-header px-2">Register your Account</h1>
+                <p class="register-paragraph txt-lg line-height px-2">Registering an account gives you access to sweet promo and events
+                    available to only registered customers</p>
+                <br>
+                <p class="register-question txt-sm px-2">Already got an acount? <a href="#">Sign in</a></p>
+            </div>
+
+            <!-- <div class="register-form">
```

Comment:
If this block of code that is commented out is not in use, kindly remove it. There is no need for having them in your code